docs/modules/user-guide/partials/proc_adding-kubernetes-component-to-a-devfile.adoc

```diff
@@ -12,116 +12,45 @@ This section describes how to add a `kubernetes` component to a devfile. A compl
-+
-.Adding `kubernetes` component using the `referenceContent` attribute
-====
-[source,yaml]
-----
-  components:
-    - name: mysql
-      kubernetes:
-        reference: petclinic.yaml
-        referenceContent: |
-             kind: List
-             items:
-              -
-               apiVersion: v1
-               kind: Pod
-               metadata:
-                name: ws
-               spec:
-                containers:
-               ... etc
-----
-====
-
-. Overriding container entrypoints. As with the xref:component-type-container{context}[`container` component], it is possible to override the entrypoint of the containers contained in the Kubernetes or OpenShift list using the `command` and `args` properties (as link:https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#notes[understood] by Kubernetes).
 +
-There can be more containers in the list (contained in Pods or Pod templates of deployments). It is therefore necessary to select which containers to apply the entrypoint changes to as follows:
-+
-.Overriding container entrypoints
+.Adding `kubernetes` component using the `inlined` property
 ====
 [source,yaml]
 ----
   components:
-    - name: appDeployment
-      kubernetes:
-        reference: app-deployment.yaml
-        entrypoints:
-        - parentName: mysqlServer
-          command: ['sleep']
-          args: ['infinity']
-        - parentSelector:
-          app: prometheus
-          args: ['-f', '/opt/app/prometheus-config.yaml']
+  - name: myk8deploy
+    kubernetes:
+      inlined: |
+        apiVersion: batch/v1
+        kind: Job
+        metadata:
+          name: pi
+        spec:
+          template:
+            spec:
+              containers:
+              - name: job
+                image: myimage
+                command: ["some",  "command"]
+              restartPolicy: Never
 ----
 ====
 +
-The `entrypoints` list contains constraints for picking the containers along with the `command` and `args` parameters to apply to them. In the example above, the constraint is `parentName: mysqlServer`, which will cause the command to be applied to all containers defined in any parent object called `mysqlServer`. The parent object is assumed to be a top level object in the list defined in the referenced file, which is `app-deployment.yaml` in the example above.
-+
-Other types of constraints (and their combinations) are possible:
-+
-`containerName`:: the name of the container
-`parentName`:: the name of the parent object that (indirectly) contains the containers to override
-`parentSelector`:: the set of labels the parent object needs to have
-+
-A combination of these constraints can be used to precisely locate the containers inside the referenced Kubernetes list.
+. `Kubernetes` or `openshift` components allow you to specify the endpoint through the endpoint property.
```

Comment:
IBM SG: " allow verb
Avoid stating that inanimate objects grant abilities to people, as in "the product allows you to...". Whenever possible, use a direct, user-focused alternative such as "you can use the product to..." or "with this product, users can..." (or replace it with "enables you to").